README.md

```diff
@@ -1,181 +1,25 @@
-# gremtune
+# gremcos
 
-[![GoDoc](http://img.shields.io/badge/godoc-reference-blue.svg)](http://godoc.org/github.com/schwartzmx/gremtune) [![Build Status](https://travis-ci.org/schwartzmx/gremtune.svg?branch=master)](https://travis-ci.org/schwartzmx/gremtune) [![Go Report Card](https://goreportcard.com/badge/github.com/schwartzmx/gremtune)](https://goreportcard.com/report/github.com/schwartzmx/gremtune)
+[![GoDoc](https://godoc.org/github.com/supplyon/gremcos?status.svg)](https://godoc.org/github.com/supplyon/gremcos) ![build](https://github.com/supplyon/gremcos/workflows/build/badge.svg?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/supplyon/gremcos)](https://goreportcard.com/report/github.com/supplyon/gremcos)
 
-- [ ] **Adjust README --> go-gremlin-cosmos**
+Gremcos is a fork of [schwartzmx/gremtune](https://github.com/schwartzmx/gremtune) with alterations to make it compatible with [Azure Cosmos](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) which is a Graph Database (Gremlin API) for Azure.
```

Comment:
we make it compatible to the "Gremlin API of Azure Cosmos DB". Cosmos offers more than just the graph part, that needs to be clear here.

Reply:
fixed